src/js/entry.js

```diff
@@ -128,27 +128,29 @@ export function chromeInit(libjwt) {
 }
 
 export function bootstrap(libjwt, initFunc) {
+    const getUser = () => {
+        // here we need to init the qe plugin
+        // the "contract" is we will do this before anyone
+        // calls/finishes getUser
+        // this only does something if the correct localstorage
+        // vars are set
+
+        qe.init();
+
+        return libjwt.initPromise
+        .then(libjwt.jwt.getUserInfo)
+        .catch(() => {
+            libjwt.jwt.logoutAllTabs();
+        });
+    };
+    const fetchPermissions = createFetchPermissionsWatcher(getUser, libjwt);
```

Comment:
I'd change this to consume only cahce
```JS
const fetchPermissions = getUser().then(() => createFetchPermissionsWatcher(cache));
```

Reply:
In order to make it work correctly you'd have to moce `let chromeCache` on line 56 move above chrome bootstrap.